### PR TITLE
13.13 CIFAR-10

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by goldmermaid** [comment]
Don't forget to create and update the forum links.

**Comment by mli** [comment]
Job d2l-zh/PR-806/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-806/

**Comment by mli** [comment]
Job d2l-zh/PR-806/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-806/

**Comment by goldmermaid** [comment]
`numref`, `cite` check done

**Comment by mli** [comment]
Job d2l-zh/PR-806/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-806/

File: chapter_computer-vision/kaggle-cifar10.md [hunk 1]
```diff
@@ -430,7 +451,9 @@ def train(net, train_iter, valid_iter, num_epochs, lr, wd, devices, lr_period,
 
 ## 训练和验证模型
 
-现在，我们可以训练和验证模型。以下所有超参数都可以调整。例如，我们可以增加纪元的数量。当 `lr_period` 和 `lr_decay` 分别设置为 50 和 0.1 时，优化算法的学习速率将在每 50 个纪元后乘以 0.1。只是为了示范，我们在这里只训练一个时代。
+现在，我们可以训练和验证模型了，而以下所有超参数都可以调整。
+例如，我们可以增加周期的数量。当 `lr_period` 和 `lr_decay` 分别设置为 50 和 0.1 时，优化算法的学习速率将在每 50 个周期乘以 0.1。
+为了示范，我们在这里只训练一个周期。
```
Comment (astonzhang) [inline chapter_computer-vision/kaggle-cifar10.md:456]: 为了示范 -> 为便于演示

Comment (goldmermaid) [reply chapter_computer-vision/kaggle-cifar10.md:456]: I saw you added [here](https://github.com/d2l-ai/d2l-zh/commit/8d9de7107c6fc550a0e7d7fbbe6cf6353f3421e4). Thanks!

**Pull request merged**